tests/ui_tests/app/page/cluster_list/page.py

```diff
@@ -51,3 +58,53 @@ def get_cluster_info_from_row(self, row: int) -> dict:
             "description": self.find_child(cluster_row, row_elements.description).text,
             "state": self.find_child(cluster_row, row_elements.state).text,
         }
+
+    def click_action_btn_in_row(self, row: WebElement):
+        self.find_child(row, self.table.table.ClusterRow.actions).click()
+
+    def click_import_btn_in_row(self, row: WebElement):
+        self.find_child(row, self.table.table.ClusterRow.imports).click()
+
+    @allure.step("Run action {action_name} for cluster")
+    def run_action_in_cluster_row(self, row: WebElement, action_name: str):
+        self.click_action_btn_in_row(row)
+        self.wait_element_visible(self.table.table.ActionPopup.block)
+        self.find_and_click(self.table.table.ActionPopup.button(action_name))
+        self.wait_element_visible(ActionDialog.body)
+        self.find_and_click(ActionDialog.run)
+
+    @contextmanager
+    def wait_cluster_state_change(self, row: WebElement):
+        state_before = self.find_child(row, self.table.table.ClusterRow.state).text
```

Comment:
`get_cluster_state_from_row` can be used here, am I right?
This way we have more predictable behavior

Reply:
thnx! changed